spaceship/lib/spaceship/connect_api/models/beta_tester.rb

```diff
@@ -9,6 +9,14 @@ class BetaTester
       attr_accessor :email
       attr_accessor :invite_type
       attr_accessor :invitation
+      attr_accessor :beta_tester_state
```

Comment:
Thanks for the contribution!

Could you update the unit test in `./spaceship/spec/connect_api/models/beta_tester_spec.rb`?

Thanks

Reply:
Does this response in `beta_tester_spec.rb` come from `spaceship/spec/connect_api/fixtures/testflight/beta_testers.json` ?

Reply:
I think so

Reply:
Done. @lacostej